src/vm/codeman.cpp

```diff
@@ -1439,6 +1439,8 @@ static void LoadAndInitializeJIT(LPCWSTR pwzJitName, OUT HINSTANCE* phJit, OUT I
 
         EX_TRY
         {
+            bool fContinueToLoadJIT = false;
+#if !defined(FEATURE_CORECLR)
```

Comment:
In the JIT, this is determined by SELF_NO_HOST. So I assume the JIT will be built with SELF_NO_HOST defined for all cases of FEATURE_CORECLR defined? But it also looks like sxsJitStartup in the JIT does nothing if SELF_NO_HOST is defined, so why is this necessary?